modules/@ergonode/attributes/src/config/extends.js

```diff
@@ -2,6 +2,99 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
+import {
+    prepareOptionsData,
+    prepareParametersData,
+    prepareTextAreaData,
+    sendOptionsData,
+    setParametersData,
+    setTextAreaData,
+} from '@Attributes/extends/methods';
+
+import {
+    Components,
+    Icons,
+    Store,
+} from './imports';
+
+const getTypeConfiguration = ({
+    $this = null, type,
```

Comment:
it's not need for assigning any default value, in default state of switch it will be returnined as null